include/openssl/crypto.h

```diff
@@ -332,6 +332,9 @@ int FIPS_mode(void);
 int FIPS_mode_set(int r);
 
 void OPENSSL_init(void);
+void OPENSSL_fork_prepare(void);
+void OPENSSL_fork_parent(void);
+void OPENSSL_fork_child(void);
```

Comment:
These need to be guarded with `#ifndef OPENSSL_SYS_VMS`.  Do a `make update` afterwards and you'll see the effect.  This is needed, or linking the shared library will complain about missing symbols.